### ISSUE TITLE
Use viewport in gui to make games with moving camera easier 

### ISSUE THREAD

**Issue by Jeremi360** [rakugoteam/Rakugo-archive#299]
Now when developer try to use camera in rakugo project this camera will move, but gui will stay in place.
So I need to make a viewport and reparent current scene to this using some thing like this:

https://godotengine.org/qa/46638/how-to-change-the-parent-of-current-node

**Comment by Jeremi360** [comment]
partially done - need to be tested when change scene.
Also mouse left click to get dialog forward stop to works - need to be fixed

**Comment by Jeremi360** [comment]
~Also mouse left click to get dialog forward stop to works - need to be fixed~ fixed.
Now it need to be tested when change scene.

**Comment by Jeremi360** [comment]
Okey it change scenes I even add loading screen,
but it looks like loaded scene don't react to any godot/rakugo signals :disappointed:
It is on branch `viewport`.

**Comment by Jeremi360** [comment]
A corrding to one comment on this I shoulde add what I expect and what I got and also small repo to make eaiser to find bug (I'm not sure if I can do that in this case).
So I expeted that:
1. Launch game and choose new game
1. I get one line of dialog
1. I click left mose button to get further
1. Scene is change to other and start new dialog

What I get:
1. Launch game and choose new game
1. I get one line of dialog
1. I click left mose button to get further
1. Scene is change to other
